app/relay.config.js

```diff
@@ -14,6 +14,6 @@ module.exports = {
     "**/__generated__/**",
     "**/server/**",
   ],
-  schema: "./server/schema.graphql",
+  schema: "./schema/schema.graphql",
```

Comment:
the new `relay-compiler` doesn't like it when the schema is in one of the excluded folders